swagger_spec_validator/common.py

```diff
@@ -19,6 +19,11 @@
 except ImportError:
     from yaml import SafeLoader
 
+try:
+    from functools import lru_cache
+except ImportError:
```

Comment:
nit. Add noqa to avoid coverage regression
```suggestion
except ImportError:  # noqa: py2
```